packages/core/test/invoke.test.ts

```diff
@@ -2030,18 +2052,17 @@ describe('invoke', () => {
         states: {
           counting: {
             invoke: {
-              src: () =>
-                fromObservable(() =>
-                  infinite$.pipe(
-                    map((value) => {
-                      if (value === 5) {
-                        throw new Error('some error');
-                      }
+              src: fromObservable(() =>
+                infinite$.pipe(
```

Comment:
other call sites dropped `infinite$` in favor of `interval(10)`. Should this be adjusted too?

Reply:
Also, note that `infinite$` is actually declared as `interval(10)`. So the whole "infinite" bit doesn't actually apply to it at all.